recipe/meta.yaml

```diff
@@ -41,33 +187,72 @@ outputs:
         - xz.exe --help  # [win]
         - unxz.exe --help  # [win]
         - if not exist %LIBRARY_PREFIX%\bin\liblzma.dll exit 1  # [win]
-        - if not exist %LIBRARY_PREFIX%\lib\liblzma.lib exit 1  # [win]
+        - if not exist %LIBRARY_PREFIX%\lib\lzma.lib exit 1  # [win]
         - if not exist %LIBRARY_PREFIX%\include\lzma.h exit 1  # [win]
         - test -f ${PREFIX}/lib/liblzma${SHLIB_EXT}  # [unix]
         - test ! -f ${PREFIX}/lib/liblzma.a          # [unix]
-  - name: xz-static
+    about:
+      license: 0BSD AND LGPL-2.1-or-later AND GPL-2.0-or-later
+      license_file:
+        - COPYING
+        - COPYING.0BSD
+        - COPYING.LGPLv2.1
+        - COPYING.GPLv2
+  - name: liblzma-static
     script: install.sh  # [unix]
     script: install.bat  # [win]
     requirements:
       build:
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+        - cmake-no-system  # [win]
+        - ninja  # [win]
+        - make  # [not win]
+      host:
+        - {{ pin_subpackage('xz', exact=True) }}
+      run:
+        - {{ pin_subpackage('xz', exact=True) }}
```

Comment:
I think it would be cleaner for `liblzma-static` to only depend on `liblzma-devel`, and not the binaries.